developers/ide/eclipse.md

```diff
@@ -5,65 +5,153 @@ title: Eclipse IDE
+   |**openHAB Development**| **Debug/Demo Environment (Required)**
+   |openHAB Add-ons        | Installs all openHAB 2 Add-ons
+   |openHAB ZigBee Binding | ZigBee Binding Development
+   |openHAB Z-Wave Binding | Z-Wave Binding Development
+   |openHAB BACNet Binding | BACNet Binding Development
+   |openHAB Web UIs        | Web UIs Development
+   |openHAB Core Framework| Core Framework Development
+
+    **NOTE: If you select `openHAB Add-ons` the openHAB2-addons repository will be checked out and all addon projects are imported in Eclipse. If you want to develop a new binding or a specific binding it is recommended to clone your own fork of openHAB2-addons and import only the projects you work on.**
+
+    **If you have selected `openHAB Add-ons` during the installation and these projects have been imported during the setup process and show up in the Package Explorer, select `OH2 Add-ons` and from right-click menu select "Close Projects": *this significantly speeds up the setup*.
+    Re-open only the binding project(s) you would like to work on.**
+
+
+
+1. Click `Next>`, verify/modify Root and install folder name. Click on `Show all variables` to open the window shown below.
+
+   ![variables](./images/ide_setup_eclipse_5_variables.png)
+
+   Explanation of some of the variables
+   - `Root install folder`: The base folder where the `Installation folder` will be placed.
+   - `Installation folder name`: This is the directory in the root install folder everything will be installed in.
+   - `Github user ID`: This is your GitHub user name used to configure the cloned git projects.
+
 
-1. Under `Github Projects > openHAB` select `openHAB Development` and any desired option from `openHAB Add-ons` (includes all add-ons from openhab2-addons repo), `openHAB ZigBee Binding` or `openHAB Z-Wave Binding`.
+1. Click `Next>` and `Finish` to start installation.
 
-   ![select projects](./images/ide_setup_eclipse_projects.png)
+   During install accept licence agreement, "Unsigned Content" for Bndtools, and Eclipse Foundation certificates when requested to complete IDE installation.
 
-1. Click `Next>`, verify/modify Root and install folder name, click `Next>` and `Finish` to start installation. 
-    During install accept licence agreement, "Unsigned Content" for Bndtools, and Eclipse Foundation certificates when requested to complete IDE installation. 
-    When the Eclipse installation is finished the IDE is launched automatically.
 
-1. **It's important**, during the first Eclipse IDE launch, to leave Eclipse open until all openHAB related initial Setup tasks / Download sources / Builds are completed. 
+1. At this point the Eclipse installer is finished and the Eclipse IDE is automatically launched to continue the installation process.
+
+   **It's important**, during the first Eclipse IDE launch, to leave Eclipse open until all openHAB related initial Setup tasks / Download sources / Builds are completed.
```

Comment:
Maybe also something for
https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers